batch/batch/server/database.py

```diff
@@ -122,15 +122,22 @@ def __init__(self, db):
     async def has_record(self, id):
         return await super().has_record({'id': id})
 
+    async def get_jobs(self, batch_id):
+        async with self._db.pool.acquire() as conn:
+            async with conn.cursor() as cursor:
+                batch_jobs_name = self._db.batch_jobs.name
+                sql = f"""SELECT * FROM `{self.name}`
```

Comment:
One more thing I noticed, isn't self.name = batch?  Shouldn't this be selecting from `jobs` since we want to return job records?

Reply:
Cotton is right.